checkpoint/recovery.go

```diff
@@ -107,13 +107,14 @@ func copyToLocalFile(ctx context.Context, logger log.Log, fs afero.Fs, dataDir,
 	if err != nil {
 		return "", fmt.Errorf("%w: parse recovery URI %v", err, uri)
 	}
-	dst := RecoveryFilename(dataDir, filepath.Base(parsed.Path), restore)
+	dst := RecoveryFilename(dataDir, filepath.Base(parsed.String()), restore)
 	if parsed.Scheme == "file" {
-		_, err = fs.Stat(parsed.Path)
+		src := filepath.Join(parsed.Host, parsed.Path)
```

Comment:
Why have you added the host here?

Reply:
On windows `parsed.Host` is the volume: `file://C:\Users\piersy` -> `parsed.Host` = `C:` `parsed.Path` = `/Users/piersy`

Reply:
This is btw. already merged to the `v0.3.10` branch, it was part of the fix for checkpoint.